### PR TITLE
feature(glam): Add non-normalized histograms on Glean metrics

### PR DESCRIPTION
**Pull request by edugfilho**
Adds non-normalized aggregations to Glean metrics.

By default, GLAM aggregates histograms by normalizing on `client_id`, which means clients have the same weight in their contribution to the histogram's values, regardless of how many times they submitted pings containing histograms.
A non-normalized aggregation is a way to aggregate histograms in which submissions (instead of clients) have the same weight. This means that clients that send more submissions will have more weight on the final aggregation.

Non-normalized aggregations were added to legacy telemetry probes in https://github.com/mozilla/bigquery-etl/pull/3873 and now I'm adding them to Glean metrics. 
While the first implementation duplicates the `bucket_counts` task, this one adds the non-normalized data to the existing tasks, which is likely to be more efficient because it will scan less data. If that proves true I'll back-port it to legacy telemetry.

Non-normalized data for Glean metrics will show up on GLAM starting from the day this gets merged. In other words, there will be no data backfill.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-3934)

### CODE REVIEW

File: bigquery_etl/glam/templates/extract_probe_counts_v1.sql [hunk 1]
```diff
@@ -17,6 +17,8 @@ WITH final_probe_extract AS ( SELECT
     MAX(total_users) as total_users,
     MAX(IF(agg_type = "histogram", mozfun.glam.histogram_cast_json(aggregates), NULL)) as histogram,
     MAX(IF(agg_type = "percentiles", mozfun.glam.histogram_cast_json(aggregates), NULL)) as percentiles,
+    MAX(IF(agg_type = "histogram", mozfun.glam.histogram_cast_json(non_norm_aggregates), NULL)) as non_norm_histogram,
```
Comment (edugfilho) [inline bigquery_etl/glam/templates/extract_probe_counts_v1.sql:20]: This is the last step of the ETL. The new columns `non_norm_histogram` and `non_norm_percentiles` are added the way `histogram` and `percentiles` are.

File: bigquery_etl/glam/templates/extract_probe_counts_v1.sql
Comment (edugfilho) [inline bigquery_etl/glam/templates/extract_probe_counts_v1.sql:1]: This is the last step of the ETL. The new columns `non_norm_histogram` and `non_norm_percentiles` are added the way `histogram` and `percentiles` are.

File: bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql
Comment (edugfilho) [inline bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql:1]: This task is where normalization happens, thus we need to add non-normalized columns. For that I wrote `mozfun.glam.histogram_normalized_sum_with_original` (also in this PR) which is a modification of `mozfun.glam.histogram_normalized_sum` that retains the original, non-normalized, histogram values.

File: bigquery_etl/glam/templates/probe_counts_v1.sql [hunk 2]
```diff
@@ -105,6 +112,9 @@ WITH probe_counts AS (
   windows_probe_counts AS (
     SELECT
       *
+      {% if is_scalar %}
+      , aggregates AS non_norm_aggregates
+      {% endif %}
```
Comment (BenWu) [inline bigquery_etl/glam/templates/probe_counts_v1.sql:117]: I don't think this is used since it's redefined in the query below

**Review by BenWu — APPROVED** [review]

**Pull request merged**